resolvers.ts

```diff
@@ -141,6 +236,31 @@ export const resolvers: Resolvers = {
       });
       return annotations;
     },
+    CoverageVizSummary: async (root, args, context, info) => {
+      // should be fetched using pipeline run id instead of sample id
+      // from the new backend
+      try {
+        const coverage_viz_summary = await get(
+          `/samples/${args.sampleId}/coverage_viz_summary`,
+          args,
+          context
+        );
+        const return_obj: any[] = [];
+        for (const key in coverage_viz_summary) {
+          for (const accension of coverage_viz_summary[key][
+            "best_accessions"
+          ]) {
+            return_obj.push({
+              pipeline_id: key,
+              ...accension,
+            });
+          }
+        }
+        return return_obj;
+      } catch (error) {
+        return [];
```

Comment:
I know it was asked for the `console.log` to be removed, but should we return the error still?  I'm not sure if it's different with relay, but typically graphQL responses have an errors field that is an array right?

Reply:
I looked into error handling a bit, and here's some recommended options I found

- graphQL mesh [documents error handling in custom resolvers](https://the-guild.dev/graphql/mesh/docs/guides/error-masking#custom-resolvers-error-handling)
- Relay [recommends creating types for error information as part of the schema](https://relay.dev/docs/guided-tour/rendering/error-states/#accessing-errors-in-graphql-responses)
- The graphQL spec includes a [top level `errors` entry in the response that includes a list of errors](http://spec.graphql.org/draft/#sec-Errors), as well as some other information depending on the error.

I didn't try this out, but it's possible the graphQL mesh approach ends up generating an error field that matches the graphQL spec.  This might be worth getting some other inputs on from the front end team.

Reply:
good call, let's just remove the try/catch and make it consistent with the other resolvers.

Reply:
Just to make sure, the error here would've been some TS error, not an error coming back from the API or something like that?